cosmos/tumbler.json

```
@@ -24,7 +24,7 @@
     {
       "coinDenom": "GRAIN",
       "coinMinimalDenom": "ugrain",
-      "coinDecimals": 6
+      "coinDecimals": 6,
```

The comma has to be removed here.
```suggestion
      "coinDecimals": 6
```